src/components/notice-view.js

```diff
@@ -79,6 +86,7 @@ class NoticeView extends Component {
                       'MMMM Do YYYY, h:mm:ss a'
                     )}
                   </p>
+                  {mediaPath ? <Button onClick={this.download}>Download PDF</Button> : null}
```

Comment:
Instead of downloading let the PDF be opened in a new tab. That will be more user-friendly I guess.

Reply:
Nice work Aditya! It's working as expected. We can introduce some aesthetics change in the button format though:

![image](https://user-images.githubusercontent.com/34851451/95669299-fd746700-0b9c-11eb-930e-4b01bedab641.png)
Ping @gouranshi or @KashishJagyasi for this.